documentation/docs/install-pmm/install-pmm-server/deployment-options/docker/preview_env_var.md

```diff
@@ -1,12 +1,39 @@
 # Preview environment variables
 
 !!! caution alert alert-warning "Warning"
-     The `PERCONA_TEST_*` environment variables are experimental and subject to change. It is recommended that you use these variables for testing purposes only and not on production.
+     The `PERCONA_TEST_*` environment variables are experimental and subject to change. These variables are intended for testing purposes only and should not be used in production environments.
+
+     For stable, production-ready configuration options, see the main [Environment variables for PMM Server](../docker/env_var.md) documentation.
+
+## Available preview variables
 
 | Variable                                                      | Description
 | ------------------------------------------------------------- | --------------------------------------------------------------------------
 | `PERCONA_TEST_SAAS_HOST`                                      | SaaS server hostname.
-| `PMM_CLICKHOUSE_ADDR`                                         | Name of the host and port of the external ClickHouse database instance.
-| `PMM_CLICKHOUSE_DATABASE`                                     | Database name of the external ClickHouse database instance.
-| `​​PMM_CLICKHOUSE_USER`                                         | Database username.
-| `PMM_CLICKHOUSE_PASSWORD`                                     | Database username password.
+| `PMM_CLICKHOUSE_ADDR`                                         | Name of the host and port of the external ClickHouse database instance
+| `PMM_CLICKHOUSE_DATABASE`                                     | Database name of the external ClickHouse database instance
+| `​​PMM_CLICKHOUSE_USER`                                         | Database username
+| `PMM_CLICKHOUSE_PASSWORD`                                     | Database username password
```

Comment:
```suggestion
| `PMM_CLICKHOUSE_DATABASE`                                     | Database name of the external ClickHouse instance
| `​​PMM_CLICKHOUSE_USER`                                         | Database user
| `PMM_CLICKHOUSE_PASSWORD`                                     | Database user password
```

Reply:
Thy latter two are easier to comprehend IMO if we say "user" instead of "username".